menu/reader.js

```diff
@@ -0,0 +1,36 @@
+// Import Dependencies
+const fs = require("fs");
+const YAML = require("js-yaml");
+
+class Menu {
+    constructor(text) {
+        this.delivery = text.Delivery;
+        this.instructions = text?.Instructions; // If instructions provided
```

Comment:
is the `?` supposed to be there?

Reply:
The '?' is just to indicate it's an optional field as the current menu.yml file does not contain it, but I can remove it doesn't affect the object